File: 1431. Kids With the Greatest Number of Candies/Solution.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        _max = max(candies) - extraCandies
        ans = []
        for candie in candies:
            ans.append(candie >= _max)

        return ans

File: 1431. Kids With the Greatest Number of Candies/Solution.c
bool* kidsWithCandies(int* candies, int candiesSize, int extraCandies, int* returnSize){
    int max = candies[0];

    for(int i = 1; i< candiesSize; i++){
        if(max<candies[i]) max = candies[i];
    }

    max -= extraCandies;
    *returnSize = candiesSize;

    bool* ans = malloc(sizeof(bool)* *returnSize);

    for(int i = 0;i<candiesSize;i++){
        ans[i] = candies[i]>=max;
    }

    return ans;

}